client/package.json

```diff
@@ -43,6 +43,10 @@
     "@angular/platform-browser": "^8.2.0",
     "@angular/platform-browser-dynamic": "^8.2.0",
     "@angular/router": "^8.2.0",
+    "@ngrx/effects": "^8.5.2",
```

Comment:
None of this NGRX stuff should be part of this PR

Reply:
removed